src/components/authentication/signIn.js

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Link from '@mui/material/Link';
+import Grid from '@mui/material/Grid';
+import Box from '@mui/material/Box';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+/**
+ * 
+ * @param {*} [props] 
+ * @returns {ReactComponentElement} 
+ */
+const Copyright = (props) => {
```

Comment:
I think this should be a separate component somewhere in common/shared folder. Copyright component inside signin component doesn't make much sense to me